app/helpers/teacher_sets_helper.rb

```diff
@@ -1,4 +1,61 @@
 # frozen_string_literal: true
 
 module TeacherSetsHelper
```

Comment:
There are three new methods here.  Would it be possible to add unit tests for them?

Reply:
Still Need to write more unit test cases for all the methods

Reply:
@daryanypl 100% i will write unit test cases. Right now why i did not added unit test cases is I am keep on changing the methods. Because still all functionality is not yet completed.If I complete the facets work major work will be over.Than i will write the unit test cases

Reply:
I'm a big fan of test-driven develppment.  You write a unit test, then that test acts as an outline of what the code will be doing.  You write the code method.  When code grows and progresses, you change the unit tests to match expected behavior.  Just the same way as when we write doc string as we develop, and update doc strings as we go along.